module-2/backend/week_one.md

```diff
@@ -5,28 +5,71 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+-get: sends request; finds specific URI
+-post: allows you to create a new instance in the  your data
+-delete: destroys data
+-PUT: allows you to update or replace a certain element in a database
+-PATCH: like put, it also allows you to update a certain element in a database    
+
 2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
-  ```ruby
-  get '/horses' do
-    erb :index
-  end
-  ```
-
-8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
-9. What's the purpose of ERB?
-10. Why do I need a development AND test database?
-11. What is CRUD and why is it important?
-12. What does HTTP stand for? 
-13. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
-14. What's an ORM?
-15. What's the most commonly used ORM in ruby (Sinatra & Rails)?
-16. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
-17. What's a migration? 
-18. When you create a migration, does it automatically modify your database?
-19. How does a model relate to a database?
-20. What is the difference between `#new` and `#create`?
+- Sinatra is a framework; a DSL language that allows us to manipulate code by eliminating extra work that’s automatically taken care of in the background. 
+
+3. What is MVC?
+- Model, View, Control: These are sub folders that live within our app parent folder. A framework that allows you to build apps in a manner that is convention.
+
+4. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+-There is a order of precedence, plus most programmers are used to using a specific form of programming. 
+
+5. What types of variables are accessible in our view templates without explicitly passing them?
+-instance variables
+
+6. Given the following block of code, how would I pass an instance variable count with a value of 1 to my index.erb template?
+get '/horses' do
+  erb :index
+end
+- @count = 1
+
+7. In the same code block, how would I pass a local variable name with a value of Mr. Ed to the view?
+- name = ‘Mr. Ed’ 
+- :locals => {name: name}

```

Comment:
This would be on the same line as `erb :index`, so all together:

```
get '/horses' do
  @count = 1
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```